api/v1beta2/kyma_types.go

```diff
@@ -61,10 +61,7 @@ type Module struct {
 	// Name is a unique identifier of the module.
 	// It is used to resolve a ModuleTemplate for creating a set of resources on the cluster.
 	//
-	// Name can be one of 3 kinds:
-	// - The ModuleName label value of the module-template, e.g. operator.kyma-project.io/module-name=my-module
-	// - The Name or Namespace/Name of a ModuleTemplate, e.g. my-moduletemplate or kyma-system/my-moduletemplate
-	// - The FQDN, e.g. kyma-project.io/module/my-module as located in .spec.descriptor.component.name
+	// Name can ONLY be the ModuleName label value of the module-template, e.g. operator.kyma-project.io/module-name=my-module
```

Comment:
IMO don't have the ONLY all-cap

Reply:
Also, please run `make manifests` so that the documentation change is reflected in the Kyma CRD as well